pkg/docker/runner_int_test.go

```diff
@@ -125,6 +128,9 @@ func TestRunDigested(t *testing.T) {
 
 	// prebuild default image
 	f, err = client.Build(ctx, f)
+	if err != nil {
```

Comment:
Apparently this fails. Any ideas @lkingland

Reply:
@matejvasek because registry is empty

Reply:
`f := fn.Function{Runtime: "go", Root: root, Registry: "docker.io/4141gauron3268"}` this should work

Reply:
Does that operation have any effect? What does "prebuild default image" mean?

Reply:
is `f, err = client.Build(ctx, f)` really needed?

Reply:
the point of the test is to "override" a prebuilt image with a digested image. `f build` -> `f run --image=X` since the --image should take precedence

Reply:
But the operation is seemingly NOOP. Neither the func structure nor the data on filesystem are changed. It looks like it does nothing.

Reply:
I think it just crashes.

Reply:
Maybe it would do something if we applied the changes you mentioned above?

Reply:
Yes, I think that if you add a registry (any registry) to the Function definition as David mentions, the initial build will complete.  Then, the second build will confirm that setting `.Image` will override the image which was derived during the first run.